api-reference/10 UI Components/dxChat/1 Configuration/fileUploaderOptions.md

```diff
@@ -5,7 +5,89 @@ default: null
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Configures file upload options.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+Define [uploadFile](/Documentation/ApiReference/UI_Components/dxFileUploader/Configuration/#uploadFile) in this object to enable file upload. To show the "Attach" button, pass at least an empty object to **fileUploaderOptions**.
```

Comment:
I'm not sure how this works. You link to `dxFileUploader.uploadFile`. But you say "this object", while describing `dxChat.FileUploaderOptions`. I'm confused about what you mean by "this object".